homeassistant/components/sensor/wunderground.py

```diff
@@ -630,6 +630,7 @@ def _get_attributes(rest):
 ]
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_ENTITY_NAMESPACE): cv.string,
```

Comment:
entity_namespace is used in one of wunderground test cases.
Not sure if it's correct to add it here, or if the testcase is stale?

Reply:
I think we should add it to the base platform schema. It's an option used in the entity component code, so all EntityPlatforms are affected.

Reply:
According to the [docs](https://www.home-assistant.io/docs/configuration/platform_options/):

> These options are being phased out and are only available for single platform integrations.

Reply:
Yes, but now we're talking about potentially breaking all platforms in one go. So it's needs to be a conscious decision.

Reply:
OK, fixed.